src/Package/Util/Header/Version.php

```diff
@@ -90,14 +90,14 @@ public function fileHasVersionMacro($fname)
     public function getVersionFromHeader()
     {
         $headers = glob($this->package->getSourceDir().DIRECTORY_SEPARATOR.'*.h');
-        $pat = ',define\s+'.$this->macroName.'\s+"(.*)",i';
+        $pat = ',define\s+'.$this->macroName.'\s+(".*"|.*\b),i';
```

Comment:
your regex would allow to match things like ``"1.2.3`` or ``1.2.3"`` or many other things. IMO, the regex need to be stricter about what it matches than just ``.*``

@Hywan there is another bug btw: the macro name should be escaped with ``preg_quote`` when inserting it into the regex, to avoid issues with chars with a special meaning